modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp

```diff
@@ -73,22 +73,60 @@ long cpDefinitionId = cpCatalogEntry.getCPDefinitionId();
 					<aui:button cssClass="btn btn-primary btn-sm my-2" href="<%= cpContentHelper.getReplacementCommerceProductFriendlyURL(cpSku, themeDisplay) %>" value="replacement-product" />
 				</c:if>
 
-				<c:if test="<%= (cpSku != null) && (cpSku.getDiscontinuedDate() != null) %>">
-
-					<%
-					Format format = FastDateFormatFactoryUtil.getSimpleDateFormat("MMMMM dd, yyyy", locale, timeZone);
-					%>
+				<%
+				Format dateFormat = FastDateFormatFactoryUtil.getSimpleDateFormat("MMMMM dd, yyyy", locale, timeZone);
+				%>
 
+				<c:if test="<%= (cpSku != null) && (cpSku.getDiscontinuedDate() != null) %>">
 					<p class="my-2">
 						<span class="font-weight-semi-bold">
 							<%= LanguageUtil.get(request, "end-of-life") %>
 						</span>
 						<span>
-							<%= format.format(cpSku.getDiscontinuedDate()) %>
+							<%= dateFormat.format(cpSku.getDiscontinuedDate()) %>
 						</span>
 					</p>
 				</c:if>
 
+				<c:if test="<%= cpSku != null %>">
+					<div>
+						<% List<CommerceInventoryReplenishmentItem> commerceInventoryReplenishmentItems =
+							cpContentHelper.getOrderedIncomingQuantity(request, cpSku.getSku()); %>
+
+						<c:if test="<%= commerceInventoryReplenishmentItems.size() > 0 %>">
+							<div class="align-items-center d-flex">
+								<p class="m-0"><%= cpContentHelper.getOrderedIncomingQuantityLabel(commerceInventoryReplenishmentItems.get(0), dateFormat, request) %></p>
+
+								<%
+								boolean isMultipleIncomingQuantity = commerceInventoryReplenishmentItems.size() > 1;
+
+								StringBundler toolTipContentSB = new StringBundler("");
```

Comment:
I believe this should give the same results.

`StringBundler toolTipContentSB = new StringBundler();`

If not use
`StringBundler toolTipContentSB = new StringBundler(StringPool.BLANK);`

Reply:
Exactly, please use `StringBundler toolTipContentSB = new StringBundler();`